### PR TITLE
Feature/forcingsanity

### PR DESCRIPTION
**Pull request by hyunchul386**
## Purpose
Adding a sanity check tool for DATM forcing in python and yaml

## Description
Followings are added
forcing_datm_sanity.yaml
forcing_datm_sanity.py

### CODE REVIEW

File: forc/SanityCheck/forcing_datm_sanity.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+                           #hyun-chul.lee@noaa.gov
+import matplotlib
+#matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import matplotlib.colors as mcolors
+import netCDF4 as nc
+from netCDF4 import Dataset
```
Comment (guillaumevernieres) [inline forc/SanityCheck/forcing_datm_sanity.py:6]: not used

File: forc/SanityCheck/forcing_datm_sanity.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+import numpy as np
+import argparse
+import glob
```
Comment (guillaumevernieres) [inline forc/SanityCheck/forcing_datm_sanity.py]: not used

File: forc/SanityCheck/forcing_datm_sanity.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+import os
```
Comment (guillaumevernieres) [inline forc/SanityCheck/forcing_datm_sanity.py]: not used

File: forc/SanityCheck/forcing_datm_sanity.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+import yaml
+import sys
+
+def plot_zonal_ave(data1zave,data1zmax,data1zmin,lats1,data2zave,data2zmax,data2zmin,lats2,plotpath,v,sdate,d1nm,d2nm,ne):
```
Comment (guillaumevernieres) [inline forc/SanityCheck/forcing_datm_sanity.py:11]: ... that's a long list of arguments! Consider maybe using at least simple data structures at least to clarify your code?

File: forc/SanityCheck/forcing_datm_sanity.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+        plt.xlabel('Latitude',fontsize=6)
+
+    if (pn == 27): 
+       #plt.show()
+        fname = [plotpath+d1nm+"_"+d2nm+"_"+sdate+".png"]
+        outname = "".join(fname)
+        plt.savefig(outname)
+        plt.close()
+
+def read_2datm_var(input1, input2, v):
```
Comment (guillaumevernieres) [inline forc/SanityCheck/forcing_datm_sanity.py]: ... that's a weird way to do it! How about you make that function just read 1 variable from 1 file name and call it twice below with input1 and input2?

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
I know it's just a plotting script, but there's some twisted logic in that code. It might be good to spend some time to clean it up and also add a `README.md`.
Also, can you create an issue describing in more details what it is you are doing/plotting?

**Comment by hyunchul386** [comment]
The code was cleaned up and added READ.md

File: forc/SanityCheck/forcing_datm_sanity.py (hunk 5 above)
Comment (hyunchul386) [reply forc/SanityCheck/forcing_datm_sanity.py]: The code was cleaned up and added READ.md

File: forc/SanityCheck/forcing_datm_sanity.py (hunk 2 above)
Comment (hyunchul386) [reply forc/SanityCheck/forcing_datm_sanity.py]: cleaned up

File: forc/SanityCheck/forcing_datm_sanity.py (hunk 3 above)
Comment (hyunchul386) [reply forc/SanityCheck/forcing_datm_sanity.py]: cleaned up

File: forc/SanityCheck/forcing_datm_sanity.py (hunk 4 above)
Comment (hyunchul386) [reply forc/SanityCheck/forcing_datm_sanity.py:11]: Users doesn't need this internal structure for the plots, and I will keep it in this case of sanity check comparisons.

**Review by guillaumevernieres — APPROVED** [review]
Thanks @hyunchul386

**Pull request merged**